### PR TITLE
i18n(fr) Update `error-reference.mdx`

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### Description (required)
Update transltion for error-reference.mdx file

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

File: src/content/docs/fr/reference/error-reference.mdx [hunk 1]
```diff
@@ -4,79 +4,89 @@ i18nReady: true
...
+- [**InvalidPrerenderExport**](/fr/reference/errors/invalid-prerender-export/)<br/>Exportation de prérendu invalide.
+- [**InvalidComponentArgs**](/fr/reference/errors/invalid-component-args/)<br/>Arguments de composant non valides.
+- [**PageNumberParamNotFound**](/fr/reference/errors/page-number-param-not-found/)<br/>Le numéro de page param est introuvable.
+- [**ImageMissingAlt**](/fr/reference/errors/image-missing-alt/)<br/>Propriété alt manquante.
+- [**InvalidImageService**](/fr/reference/errors/invalid-image-service/)<br/>Erreur lors du chargement du service d'image.
+- [**MissingImageDimension**](/fr/reference/errors/missing-image-dimension/)<br/>Dimenssions d'image manquantes
+- [**UnsupportedImageFormat**](/fr/reference/errors/unsupported-image-format/)<br/>Format d'image non supporté
+- [**UnsupportedImageConversion**](/fr/reference/errors/unsupported-image-conversion/)<br/>Converion d'image non supportée.
+- [**PrerenderDynamicEndpointPathCollide**](/fr/reference/errors/prerender-dynamic-endpoint-path-collide/)<br/Le point de terminaison dynamique généré entre en collision avec une autre route.
```
Comment (dreyfus92) [inline src/content/docs/fr/reference/error-reference.mdx:43]:
```suggestion
- [**PrerenderDynamicEndpointPathCollide**](/fr/reference/errors/prerender-dynamic-endpoint-path-collide/)<br/>Le point de terminaison dynamique généré entre en collision avec une autre route.
```

File: src/content/docs/fr/reference/error-reference.mdx [hunk 2]
```diff
@@ -4,79 +4,89 @@ i18nReady: true
...
+- [**ResponseSentError**](/fr/reference/errors/response-sent-error/)<br/>Impossible d'établir la réponse.
+- [**MiddlewareNoDataOrNextCalled**](/fr/reference/errors/middleware-no-data-or-next-called/)<br/>Le middleware n'a pas retourné de réponse ou appelé `next`.
+- [**MiddlewareNotAResponse**](/fr/reference/errors/middleware-not-aresponse/)<br/>The middleware a renvoyé quelque chose qui n'est pas un objet `Response`.
+- [**LocalsNotAnObject**](/fr/reference/errors/locals-not-an-object/)<br/>La valeur attribuée à `locals` est refusée.
+- [**LocalImageUsedWrongly**](/fr/reference/errors/local-image-used-wrongly/)<br/>Les images importées par ESM doivent être transmises telles quelles.
+- [**AstroGlobUsedOutside**](/fr/reference/errors/astro-glob-used-outside/)<br/>Astro.glob() utilisé en dehors d'un fichier Astro
+- [**AstroGlobNoMatch**](/fr/reference/errors/astro-glob-no-match/)<br/>Astro.glob() ne correspond à aucun fichier.
+- [**RedirectWithNoLocation**](/fr/reference/errors/redirect-with-no-location/)<br/Une redirection doit recevoir un emplacement avec l'en-tête `Location`.
```
Comment (dreyfus92) [inline src/content/docs/fr/reference/error-reference.mdx:57]:
```suggestion
- [**RedirectWithNoLocation**](/fr/reference/errors/redirect-with-no-location/)<br/>Une redirection doit recevoir un emplacement avec l'en-tête `Location`.
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
encore 😅

**Comment by McFlyPartages** [comment]
LGTM

**Review by thomasbnt — APPROVED** [review]
LGTM! 🚀

**Comment by dreyfus92** [comment]
LGTM les gars 🫡

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🚀

**Pull request merged**